nutils/function.py

```diff
@@ -761,7 +779,7 @@ def partial_derivative(cls, iarg: int, *args: Any) -> IntoArray:
             The partial derivative of this function to the given argument.
         '''
 
-        raise NotImplementedError('The partial derivative of {} to argument {} (counting from 0) is not defined.'.format(cls.__name__, iarg))  # pragma: nocover
+        raise NotImplementedError('The partial derivative to argument {} (counting from 0) is not defined.'.format(cls.__name__, iarg))  # pragma: nocover
```

Comment:
I see only one set of {} and two format arguments. Also, let's switch to a format string?